ts/data/project.ts

```diff
@@ -139,12 +153,31 @@ export const projectDefinition: TableDefinition<ProjectSummary, void> = {
         title: "Report Finalized",
         addParentContents(projectSummary, fragment) {
           fragment.appendChild(
-            document.createTextNode(
-              projectSummary.finalReportCompletedCount.toString()
+            displayFilteredProject(
+              projectSummary.finalReportCompletedCount,
+              projectSummary.name,
+              "Final Report"
             )
           );
         },
       },
     ];
   },
 };
+
+const completedFilterKey = "COMPLETED";
+
+function displayFilteredProject(
+  count: number,
+  projectName: string,
+  filterValue: string
+) {
+  var url = new URL(document.location.href);
+  var params = url.searchParams;
```

Comment:
* You can construct a `new URLSearchParams()` without creating a `URL` first
* Use `const` instead of `var`